qtaf-xray-plugin/src/test/java/de/qytera/qtaf/xray/config/XrayConfigHelperTest.java

```diff
@@ -16,4 +23,12 @@ public void testService() {
         configMap.setString(XrayConfigHelper.XRAY_SERVICE_SELECTOR, "cloud");
         Assert.assertEquals(XrayConfigHelper.getXrayService(), "cloud");
     }
+
```

Comment:
Please add a comment to the method

Reply:
Fixed in [6378a3c](https://github.com/Qytera-Gmbh/QTAF/pull/85/commits/6378a3cac4571064d8657cb29ba2ecfa3e0589e2).